lighthouse-cli/test/smokehouse/test-definitions/tricky-metrics/expectations.js

```diff
@@ -17,10 +17,11 @@ module.exports = [
       requestedUrl: 'http://localhost:10200/tricky-tti.html',
       finalUrl: 'http://localhost:10200/tricky-tti.html',
       audits: {
-        'first-cpu-idle': {
```

Comment:
on these you could add assertions for TBT if you want, but i don't think that's necessary or a  great test for that metric.

so i'm +1 on removing in this file too